doc/api/toolkits/axes_grid.rst

```diff
@@ -0,0 +1,21 @@
+.. _axes_grid-api-index:
+
+####################################
+The Matplotlib axes_grid Toolkit API
+####################################
+
+:Release: |version|
+:Date: |today|
+
+.. currentmodule:: mpl_toolkits
+
+Axes Grid
+---------
+
+.. autosummary::
+   :toctree: ../_as_gen
+   :template: automodule.rst
+
+   axes_grid
```

Comment:
My opinion is to get rid of `axes_grid` in the documentation if you want people to stop using it and `axes_grid1` has been around for a while. I can say at least from personal experience it's super confusing what the difference between the two is...